Makefile

```diff
@@ -368,12 +379,13 @@ lint: server/static/files.go $(GOPATH)/bin/golangci-lint
 	rm -Rf vendor
 	# Tidy Go modules
 	go mod tidy
+
 	# Lint Go files
 	$(GOPATH)/bin/golangci-lint run --fix --verbose --concurrency 4 --timeout 5m
-	# Lint UI files
-ifeq ($(STATIC_FILES),true)
-	yarn --cwd ui lint
-endif
+
+	# Lint swagger files
+	swagger validate api/openapi-spec/swagger.json
+	go test ./api/openapi-spec
```

Comment:
can we remove this line - covered by `make test`?

Reply:
Will remove